src/avatar/lib/avatar.scss

```diff
@@ -0,0 +1,22 @@
+.tbds-avatar {
+  --border-radius: 3px;
+  --size: 3rem;
```

Comment:
Is this the pattern you were suggesting in https://github.com/thoughtbot/design-system/pull/71#discussion_r275511258 ?

Reply:
We could use CSS variables in that scenario, though Sass variables would be fine, too. Mostly my point was just offering it as a configuration point, so a consumer could easily say “I need my icons to be `1.5em`, so I will override this variable to do that”

Your question is valid, though, and we should document why and when to use Sass variables, and CSS variables.